CHANGELOG.md

```diff
@@ -2,13 +2,20 @@
 
 ## 1.27.0 (TBD)
 
+### Snowpark Python API Updates
+
+#### New Features
+
+- Added support for `snowflake_cortex_sentiment`.
+
 ### Snowpark pandas API Updates
 
 #### New Features
 
 - Added support for `Series.str.ljust` and `Series.str.rjust`.
 - Added support for `Series.str.center`.
 - Added support for `Series.str.pad`.
+- Added support for applying Snowpark Python function `snowflake_cortex_sentiment`.
```

Comment:
```suggestion
- Added support for function `snowflake_cortex_sentiment` in `functions.py`.
```

Reply:
@sfc-gh-yuwang Did you mean to update the changelog for Snowpark Python API Updates or pandas API Updates? For pandas, we may want to keep the reference to `apply`

Reply:
I think the changelog for snowpark python api should be updated to what yuyang suggests here.

Reply:
Makes sense, it should be updated now!